templates/default/chef-guard.service.erb

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Chef Guard daemon
+After=network.target auditd.service
+
+[Service]
+ExecStart=/opt/chef-guard/chef-guard
+ExecReload=/bin/kill -HUP $MAINPID
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
There are some hard coded paths in the template with should be using attributes. Have a look at https://github.com/xanzy/chef-guard-cookbook/blob/master/templates/default/chef-guard.upstart.erb to see how to do this.

Also please make sure the template has the same kind of header so you can see this file is manages by Chef.